liquibase-standard/src/main/java/liquibase/command/core/UpdateToTagCommandStep.java

```diff
@@ -174,11 +174,7 @@ protected void customMdcLogging(CommandScope commandScope) {
     }
 
     @Override
-    public void postUpdateLog(int rowsAffected) {
-        if (rowsAffected > -1) {
-            Scope.getCurrentScope().getUI().sendMessage(String.format(coreBundle.getString("update.to.tag.successful.with.row.count"), rowsAffected));
```

Comment:
Update-to-tag has been replaced to Update has been successful as noticed by @rberezen